Tests/test_api/test_main_view.py

```diff
@@ -0,0 +1,26 @@
+import json
+import unittest
+
+from Tests.test_api.base import RestAPITestBase
+from kalliope._version import version_str
+
+
+class TestMainView(RestAPITestBase):
+
+    def test_server_is_up_and_running(self):
+        # response = urllib2.urlopen(self.get_server_url())
+        response = self.client.get(self.get_server_url())
+        self.assertEqual(response.status_code, 200)
```

Comment:
This test could be part of the `RestAPITestBase` ?

Reply:
RestAPITestBase is not a "test class". It doesn't start with "Test". It's more like a tool class.